extensions/funqy/funqy-knative/runtime/src/main/java/io/quarkus/funqy/runtime/bindings/knative/VertxRequestHandler.java

```diff
@@ -45,22 +42,44 @@
     protected final CurrentIdentityAssociation association;
     protected final CurrentVertxRequest currentVertxRequest;
     protected final Executor executor;
+    protected final String ceSource;
+    protected final String ceType;
 
     public VertxRequestHandler(Vertx vertx,
             BeanContainer beanContainer,
             FunctionInvoker invoker,
             ObjectMapper mapper,
-            Executor executor) {
+            Executor executor,
+            FunqyCloudEventsConfig funqyCloudEventsConfig) {
         this.vertx = vertx;
         this.beanContainer = beanContainer;
         this.invoker = invoker;
         this.executor = executor;
         this.mapper = mapper;
+
+        ceSource = or(
```

Comment:
@lance are these fallback defaults OK?

Reply:
It seems strange to me that `K_SERVICE` env var and `invoker.getName()` are doing double duty here and can either be the source or the type. I think perhaps source and type should be overridden using different env vars (`K_SERVICE` and `K_SERVICE_TYPE` maybe).

Reply:
> It seems strange to me that `K_SERVICE` env var and `invoker.getName()` are doing double duty here and can either be the source or the type. I think perhaps source and type should be overridden using different env vars (`K_SERVICE` and `K_SERVICE_TYPE` maybe).

I'v chosen `K_SERVICE` because it is set by default in a pod owned be a `ksvc`, but I can change it to `K_SERVICE_TYPE` for ce-type.
re: invoker.getName(), maybe for a ce-type we could use something like: `invoker.getName() + "/" + invoker.getOutputType().getName()`? (getName() returns fully qualified name)